cernopendata/modules/fixtures/data/terms/terms.json

```diff
@@ -12746,5 +12746,328 @@
     "type": {
       "primary": "Glossary"
     }
+  },
+  {
+    "anchor": "CERNLIB",
+    "category": "generic",
+    "definition": "CERN Program library. The CERN Program Library is a large collection of general purpose libraries and modules maintained and offered in both source and object code form on the CERN central computers. The two most popular applications based on CERNLIB are PAW and GEANT 3.21. Most of these programs were developed at CERN and are therefore oriented towards the needs of a physics research laboratory that is general mathematics, data analysis, detectors simulation, data-handling etc... applicable to a wide range of problems. ",
```

Comment:
For consistency with the reference to PAW, what about dropping the`3.21` ( or at least the `.21`) ?